src/App.css

```diff
@@ -49,6 +49,15 @@ textarea {
    font-weight: bold;
 }
 
+.metriq-navbar h2 {
+  margin: 0 10px;
+
```

Comment:
Also, please remove this line of white space.

Reply:
No problem I'll make these adjustments for you.